python/test/ngraph/test_ops_unary.py

```diff
@@ -31,7 +30,7 @@ def test_unary_op_array_err(ng_api_fn, numpy_fn, range_start, range_end):
     expected = numpy_fn(input_data)
 
     result = run_op_numeric_data(input_data, ng_api_fn)
-    assert np.allclose(result, expected)
+    np.testing.assert_allclose(result, expected, rtol=0.001)
```

Comment:
Please pass these keyword arguments: `rtol=1e-05, atol=1e-08, equal_nan=False`

Reply:
If we want to pass these arguments in every test, perhaps we should make our own custom assert function.